src/main/java/kb/weather/location/LocationService.java

```diff
@@ -0,0 +1,43 @@
+package kb.weather.location;
+
+import kb.weather.exceptions.EmptyInputException;
+import kb.weather.exceptions.ValueOutOfRangeException;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Component;
+
+@Component
+@RequiredArgsConstructor
+public class LocationService {
+    private final Double NORTH_LIMIT = 90.0;
+    private final Double SOUTH_LIMIT = -90.0;
+    private final Double EAST_LIMIT = 180.0;
+    private final Double WEST_LIMIT = -180.0;
+
+    private final LocationRepository locationRepository;
+
+    Location createLocation(String city, String region, String country, Double latitude, Double longitude) {
+        if (city.isEmpty()) {
```

Comment:
consider using `commons-lang3` library and `StringUtils.isBlank()` method

http://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html
https://mvnrepository.com/artifact/org.apache.commons/commons-lang3

Reply:
`isBlank` vs `isEmpty`
https://stackoverflow.com/questions/51299126/difference-between-isempty-and-isblank-method-in-java-11